src/pb_view.cpp

```diff
@@ -355,4 +351,26 @@ void pb_view::set_dllist_keymap_hint()
 	dllist_form.set("help", keymap_hint);
 }
 
+std::string pb_view::format_line(const std::string& podlist_format,
+	const download* dl,
+	unsigned int pos,
+	unsigned int width)
+{
+	fmtstr_formatter fmt;
+
+	fmt.register_fmt('i', strprintf::fmt("%u", pos + 1));
+	fmt.register_fmt('d',
+		strprintf::fmt("%.1f", dl->current_size() / (1024 * 1024)));
+	fmt.register_fmt(
+		't', strprintf::fmt("%.1f", dl->total_size() / (1024 * 1024)));
+	fmt.register_fmt('p', strprintf::fmt("%.1f", dl->percents_finished()));
+	fmt.register_fmt('k', strprintf::fmt("%.1f", dl->kbps()));
```

Comment:
Is the change from 2 decimal places to 1 intentional, or is it simply a copy-and-paste typo?

Reply:
Copy paste typo, I'll fix that.